invenio_oauthclient/contrib/globus.py

```diff
@@ -230,7 +230,8 @@ def account_info(remote, resp):
             'email': info['email'],
             'profile': {
                 'username': info['username'],
-                'full_name': info['name']
+                'full_name': info['name'],
+                'affiliation': ''
```

Comment:
is it still needed to hardcode to empty string ?

Reply:
yes, nice catch!